buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/PerformanceTest.java

```diff
@@ -43,7 +43,10 @@
  */
 @CacheableTask
 public class PerformanceTest extends DistributionTest {
-    private Property<String> baselinesProperty;
+    private static final String DEFAULT_BASELINE = "defaults";
+    private static final String FORCE_DEFAULT_BASELINE = "force-defaults";
+    private Property<String> configuredBaselines = getProject().getObjects().property(String.class).value(DEFAULT_BASELINE);
+    private Property<String> forkPointCommitBaseline = getProject().getObjects().property(String.class);
```

Comment:
I would expect to have a property `determinedBaselines`, which is always set by the `determineBaselines` task.

Reply:
Done.